relay-server/src/services/buffer/mod.rs

```diff
@@ -466,41 +466,46 @@ mod tests {
 
     use super::*;
 
-    fn buffer_service() -> (
+    fn mock_envelope_buffer_service(
+        config_json: Option<serde_json::Value>,
+        global_config_status: global_config::Status,
+    ) -> (
```

Comment:
While we're at it, should we make the return value of this function a little struct with named fields?

Reply:
I was considering that, will do!